lib/galaxy/tools/duplicate_file_to_collection.xml

```diff
@@ -9,9 +9,9 @@
     <!-- <edam_operations>
     </edam_operations> -->
     <inputs>
-        <param type="dataset" name="input" label="Input Dataset" help="A dataset which will be duplicated."/>
-        <param type="integer" name="number" label="Size of output collection" help="Number of times to duplicate the input dataset."/>
-        <param type="text" name="element_identifier" label="Element identifier" help="Each element in the output collection will be assigned this identifier, with an integer appended." />
+        <param name="input" type="data" label="Input Dataset" help="A dataset which will be duplicated."/>
+        <param name="number" type="integer" value="" label="Size of output collection" help="Number of times to duplicate the input dataset."/>
+        <param name="element_identifier" type="text" label="Element identifier" help="Each element in the output collection will be assigned this identifier, with an integer appended." />
```

Comment:
type_source doesn't make sense either, should be list, i think.

Reply:
Thanks, fixed that and a few more issues, now the tool works!